lib/cell_tool.ex

```diff
@@ -34,9 +34,12 @@ defmodule CellTool do
       cell: /jrtp/sys/firmware/current -> ok
   """
 
+  @cell_tool_version Mix.Project.config[:version]
+
   @doc "Function that gets call when run as CLI"
   def main(args) do
-    args |> parse_args
+    args
+      |> parse_args
```

Comment:
I noticed this earlier where here you are using the indentation for the pipe operator. but (continued below)